docs/protocol/index.md

```diff
@@ -2,133 +2,90 @@
 sidebar_position: 0
 ---
 
-# Architecture
+# Technical Architecture
 
-Learn how Evmos's architecture leverages the Cosmos SDK Proof-of-Stake functionality, EVM compatibility and fast-finality
- from Tendermint Core's BFT consensus.
+Evmos is a scalable, Proof-of-Stake blockchain that is fully compatible and
+interoperable with the Ethereum Virtual Machine (EVM). It is built using the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk/)
+which runs on top of the [Tendermint Core](https://github.com/tendermint/tendermint) consensus engine,
+to accomplish fast finality, high transaction throughput and short block times (~2 seconds).
+
+This architecture allows users to perform both Cosmos and EVM formatted transactions with Fast Finality and High throughput,
+developers to scale EVM dApps cross-chain via [IBC](https://cosmos.network/ibc)
+and tokens and assets in the network to come from different independent sources.
+
+Evmos enables these key features by:
+
+* Leveraging [modules](https://docs.cosmos.network/main/building-modules/intro.html) and other mechanisms implemented by the [Cosmos SDK](https://docs.cosmos.network/).
+* Implementing Tendermint Core's Application Blockchain Interface ([ABCI](https://docs.tendermint.com/master/spec/abci/)) to manage the blockchain.
+* Utilizing [`geth`](https://github.com/ethereum/go-ethereum) as a library to promote code reuse and improve maintainability.
+* Exposing a fully compatible Web3 [JSON-RPC](./apis/ethereum-JSON-RPC/JSON-RPC) layer for interacting with existing Ethereum clients and tooling (Metamask, Remix, Truffle, etc).
+
+The sum of these features allows developers to leverage existing Ethereum ecosystem tooling and
+software to seamlessly deploy smart contracts which interact with the rest of the Cosmos
+[ecosystem](https://cosmos.network/ecosystem).
+
+<!-- TODO ## High level Systems Design (Add visuilization) -->
 
 ## Cosmos SDK
 
 Evmos enables the full composability and modularity of the [Cosmos SDK](https://docs.cosmos.network/).
+As a Cosmos chain, Evmos is a sovereign blockchain with its own native token,
+that can connect to other chains through IBC. It includes standard modules from the SDK,
+that work side to side with Evmos-specific modules, build by the Evmos core development team.
+Check out the [list of modules](modules/index.md) to get an overview of what each module is responsible for.
 
-## Tendermint Core & the Application Blockchain Interface (ABCI)
+## Tendermint Core & ABCI
```

Comment:
Nit, Can you update this section to reference CometBFT given that Tendendermint Core is not longer maintained?